### ISSUE TITLE
Mouse not declared 

### ISSUE THREAD

**Issue by kaeptnkrunch** [nshadov/screensaver-mouse-jiggler#5]
Thanks for providing this Script. 

But when I Compile the Script there comes this issue:

```
'Mouse' was not declared in this scope Mouse.begin();
```

**Comment by dolzenko** [comment]
Was this ever working before? Probably some old version of Arduino IDE is needed?

**Comment by netomarchiori** [comment]
yeah, I'm seeing the exact same issue. Seems my board does not support Mouse related functionalities

**Comment by frijsdijk** [comment]
List of boards where Mouse.h is available:

https://www.arduino.cc/reference/en/language/functions/usb/mouse/
